src/main/java/org/folio/rest/impl/PiecesAPI.java

```diff
@@ -28,9 +62,33 @@ public void getOrdersStoragePieces(String query, String totalRecords, int offset
 
   @Override
   @Validate
-  public void postOrdersStoragePieces(org.folio.rest.jaxrs.model.Piece entity, Map<String, String> okapiHeaders,
+  public void postOrdersStoragePieces(Piece entity, Map<String, String> okapiHeaders,
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    PgUtil.post(PIECES_TABLE, entity, okapiHeaders, vertxContext, PostOrdersStoragePiecesResponse.class, asyncResultHandler);
+    pgClient.withTrans(conn -> createPiece(conn, entity)
+        .compose(ignore -> auditOutboxService.savePieceOutboxLog(conn, entity, PieceAuditEvent.Action.CREATE, okapiHeaders)))
```

Comment:
I think lets follow the same formatting as in other controllers, with 2 spaces instead of 4

Reply:
For me IDEA does 2 tabs here so if default tabbing were 4 spaces then here it will be 8 spaces. Can you please double check?

Reply:
Commited this formatting change, resolving comment